tests/test_diffraction_objects.py

```diff
@@ -319,8 +303,25 @@ def test_scale_to(org_do_args, target_do_args, scale_inputs, expected):
                 "d": None,
                 "offset": 0,
             },
+            {"xtype": "q", "yarray": np.array([10, 20, 30])},
         ),
-        (  # C2: tth and d both provided, expect ValueErrort
+    ],
+)
+def test_scale_to(org_do_args, target_do_args, scale_inputs, expected):
+    original_do = DiffractionObject(**org_do_args)
+    target_do = DiffractionObject(**target_do_args)
+    scaled_do = original_do.scale_to(
+        target_do, q=scale_inputs["q"], tth=scale_inputs["tth"], d=scale_inputs["d"], offset=scale_inputs["offset"]
```

Comment:
I think we need to fgure out a way only pass in what is needed, rather than passing in everything with some things set to default values.  It may need more test functions written, but see if you can figure out how to do it with unpacking kwargs that are handed in.